access_control.py

```diff
@@ -0,0 +1,225 @@
+"""
+This script will process the `grant_access/` and `revoke_access/` requests.
+
+Main logic lives in conda-smithy. This is just a wrapper for admin-requests infra.
+"""
+import os
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+import requests
+from ruamel.yaml import YAML
+
+GH_ORG = os.environ.get("GH_ORG", "conda-forge")
+
+CIRUN_FILENAME_RESOURCE_MAPPING = {
+    "cirun-gpu-runner": {
+        "resource": "cirun-gpu-runner",
+    },
+    "cirun-gpu-runner-pr": {
+        "resource": "cirun-gpu-runner",
+        "policy_args": ["pull_request"]
+    }
+}
+
+
+ACCESS_YAML_FILENAME = ".access_control.yml"
+
+
+def _get_access_control_files(path):
+    access_path = Path(path)
+    all_files = list(access_path.glob("*"))
+    return (
+        [f for f in all_files if f.parts[-1] != "example.txt"]
+    )
+
+
+def _get_filename_feedstock_mapping(path):
+    access_files = _get_access_control_files(path)
+    file_name_feedstock_mapping = {}
+    for file_path in access_files:
+        with open(file_path, "r") as f:
+            feedstocks = f.readlines()
+            feedstocks = [feedstock.strip() for feedstock in feedstocks]
+            file_name = file_path.parts[-1].strip(".txt")
+            file_name_feedstock_mapping[file_name] = feedstocks
+    return file_name_feedstock_mapping
+
+
+def _process_access_control_requests(path, remove=True):
```

Comment:
Why `remove=True`? I would expect requests to _grant_ access be more common than those _revoking_. I'd change the default to False.